test/functional/users_controller_test.rb

```diff
@@ -309,6 +309,20 @@ def setup
     assert_equal I18n.t('users_controller.no_user_found'), flash[:error]
   end
 
+  test 'should get username in alphabetical order' do
+    get :list, params: { sort: 'username' }
+
+    assert_response :success
+    assert assigns(:users).each_cons(2).all?{|i,j| "j.node_revisions.username" >= "i.node_revisions.username" }
+  end
+
+  test 'should sort based on last activity in decreasing order' do
+    get :list, params: { sort: 'last_activity' }
+
+    assert_response :success
+    assert assigns(:users).each_cons(2).all?{|i,j| i.updated_at >= j.updated_at }
```

Comment:
the test looks amiss though, because we want to order by node_revisions last updated right?
but this looks like its looking at users table last updated_at ..I will pull locally and try to find a work around and revert

Reply:
Hey @cesswairimu @Manasa2850 any more activity here? please let me know how I can help :sweat_smile: I see the tests are now passing

Reply:
oh, the test was a bit off, might need to rewrite... this skipped my mind...will check it out soon...thanks @Tlazypanda :heart:

Reply:
I believe @Manasa2850 is overloaded a bit but will be back online very soon, so this can hold for a bit! Thanks!